### ISSUE TITLE
What version goes with what?

### ISSUE THREAD

**Issue by jimgwhit** [laravel/jetstream#200]
&- Jetstream Version: all
- Jetstream Stack: Inertia / Livewire
- Laravel Version: 8.0, but all higher also
- PHP Version: 7.4
- Database Driver & Version: na

### Description: WIth all the different versions  it's hard to match what goes with what now.

Already on the forum there is confusion.  In laravel it's easy to tell what version of the framework is needed, via
```
"laravel/framework": "^8.0",
```

But okay you want to update, how do you know which version of jetstream is working with what version of laravel then add in fortify versions it becomes difficult to know.

How about please place a note in laravel's composer.json stating what version of what works with what.

I.e., when a laravel 8.2 comes, list everything that goes with it.  The problem laravel on github doesn't have a jetstream, it's only after running artisan it get's placed there.

An example I installed the new laravel ver v8.0.0, then updated to v8.0.1, ran composer update.  But that doesn't handle  jetstream, I started with v1.0.0.  But how do I know which newer version of jetstream will work with that version of laravel, I currently see no way of knowing, you have to guess.  There are already 6 versions higher than v1.0.0.

I consider it a bug because the docs aren't updated through minor release, so this information should be available somewhere.

Also jetstream composer.json doesn't state what version of laravel is required.

**Comment by driesvints** [comment]
> how do you know which version of jetstream is working with what version of laravel 

Hey @jimgwhit. You can find out about that in several ways. First of all, the release notes of Laravel 8 tell that it introduces Jetstream as of that version: https://laravel.com/docs/8.x/releases#laravel-8

Secondly, you can always examine the `composer.json` of Jetstream to find out about to which Laravel version it's compatible. As you can see, `^8.0` is listed here which means every Laravel 8.x version: https://github.com/laravel/jetstream/blob/1.x/composer.json#L19

> But how do I know which newer version of jetstream will work with that version of laravel, I currently see no way of knowing, you have to guess. There are already 6 versions higher than v1.0.0.

It seems you're a bit unfamiliar with SemVer, [our versioning policy](https://laravel.com/docs/8.x/releases#versioning-scheme). You can read up [here](https://semver.org) but the tl;dr is that every 1.x.x version of Jetstream will be compatible with every Laravel 8.x.x (unless under the odd circumstance that we have to bump a minor or patch version for bug or security fix reasons). 

Hope that explains. As a side note: in the future, please try to ask these questions on a support channel as we try to keep these issue trackers solely for tracking bugs ([see our issue templates](https://github.com/laravel/jetstream/issues/new/choose) when you open a new issue).
